app/Actions/Photo/Extensions/Checks.php

```diff
@@ -48,38 +48,48 @@ public function folderPermission($folder)
 	 * Check if a picture has a duplicate
 	 * We compare the checksum to the other Photos or LivePhotos.
 	 *
-	 * @return false|Photo
+	 * @param string $checksum
+	 *
+	 * @return ?Photo
 	 */
-	public function get_duplicate($checksum, $photoID = null)
+	public function get_duplicate(string $checksum): ?Photo
```

Comment:
I am really unsure about the dropping the the `$photoID` condition here. Need to see where this is used.

Reply:
This method is only called at [exactly one place of the current code](https://github.com/LycheeOrg/Lychee/blob/bc918a394a995a939b6e5651746895c464370a2d/app/Actions/Photo/Create.php#L80) and the argument `$photoID` is not used. So it was dead code.